shippable.yml

```diff
@@ -16,7 +16,8 @@ matrix:
     - env: T=units/3.6/1 A_REV=devel
     - env: T=units/3.7/1 A_REV=devel
     - env: T=units/3.8/1 A_REV=devel
-    - env: T=units/3.9/1 A_REV=devel
+# Until boto/botocore#2002 is fixed
```

Comment:
Sorry to be a pain but as you noted on irc upstream boto has merged the fix for this.  Testing this locally on 3.9 is passing now, can you back out this change?

Reply:
@jillr I knew it had been merged, I hadn't expected a release that quickly!